apollo-compiler/api/apollo-compiler.api

```diff
@@ -1,3 +1,9 @@
+public final class com/apollographql/apollo3/compiler/-checkApolloReservedEnumValueNames {
+}
+
+public final class com/apollographql/apollo3/compiler/-checkCapitalizedFields {
```

Comment:
I think that's a false positive. Just filed https://github.com/Kotlin/binary-compatibility-validator/issues/87